frontend/public/components/start-guide.tsx

```diff
@@ -50,7 +64,15 @@ export const OpenShiftGettingStarted = connect(createProjectMessageStateToProps)
           </Trans>
         </p>
         {canCreateProject && (
-          <Button variant="link" onClick={() => createProjectModal({ blocking: true })}>
+          <Button
+            variant="link"
+            onClick={() =>
+              createProjectModal({
+                blocking: true,
+                onSubmit: handleSubmit,
```

Comment:
With the removal of the `onSubmit` type, you are right that this will not compile anymore.

One option might be to improve the types somewhere else, but I would like to ask you to change the import from

```tsx
import { createProjectModal } from './modals/create-namespace-modal';
```

to

```tsx
import { createProjectModal } from './modals';
```

The reason is then the modal will lazy loaded when the modal is used. Take a look into the different `createProjectModal` you import from `modal/index.ts`.

When you search for `createProjectModal` you will see that the other imports also use the `modal/index.ts` version.